api/v1alpha1/constants.go

```diff
@@ -18,4 +18,6 @@ const (
 	OperationAnnotationForceUpdateOrDeleteInProgress = "ForceUpdateOrDeleteInProgress"
 	// OperationAnnotationForceUpdateInProgress allows update of a resource even if it is in progress.
 	OperationAnnotationForceUpdateInProgress = "ForceUpdateInProgress"
+	// OperationAnnotationForceBMCReset forces a BMC reset before next operation
+	OperationAnnotationForceBMCReset = "ForceBMCReset"
```

Comment:
Can we name this contant just `ForceReset`? That way other controllers can reuse that for their needs later.